force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html

```diff
@@ -176,8 +176,10 @@
                                     </lightning-formatted-date-time>
                                 </div>
                             </td>
-                            <td key={item.nextDonation} data-id={item.recurringDonation.Id} class={tdClasses}  data-label={labels.RD_nextDonation} tabindex="-1" aria-label={item.nextDonation} role="gridcell">
-                                <div class="next-donation" data-ndid={item.recurringDonation.Id} lwc:dom="manual"></div>
```

Comment:
`lwc:dom="manual"` is considered insecure, so I changed the logic to avoid its use.